workfront_bridge/projects/update_builder.py

```diff
@@ -97,78 +97,90 @@ def check_not_none(name, value):
             check_not_none("end_datetime", self.end_datetime)
 
     def __fullfill_project_type(self):
-        '''
+        """
         @summary: Fullfill self.project_type based on the project that will be
         update (self.wf_project_id)
         @raise WFBridgeException: if the project has an unsupported project
         type to update.
-        '''
+        """
         prj = WFProject(self.wf, self.wf_project_id)
         params = prj.get_param_values()
         if "Project Type" not in params:
-            WFBrigeException("Unknown Project Type - {} is missing Project "
-                             "Type custom form field".format(prj))
+            WFBrigeException(
+                "Unknown Project Type - {} is missing Project "
+                "Type custom form field".format(prj)
+            )
         if params["Project Type"] not in self.supported_project_types:
-            WFBrigeException("Project Type {} not supported for updating "
-                             "project {}".format(prj))
+            WFBrigeException(
+                "Project Type {} not supported for updating " "project {}".format(prj)
+            )
         self.project_type = params["Project Type"]
 
     def __build_update_email_deploy(self):
-        '''
+        """
         @return: a update wf project to update an email project.
-        '''
-        prj_being_updated = WFProject(self.wf, self.wf_project_id)
+        """
+
+        def first(_list):
```

Comment:
Hi @mikeverducci , this function is a little bit tricky but it works.

```
>>> def first(_list):
...             return _list[0] if _list else None
... 
>>> x = first([3])
>>> x
3
>>> x = first([])
>>> x
>>> x = first(None)
>>> x
>>> 
```
I was expecting an index error when _list was empty but it does not happen. Thanks for showing me that :+1:

Reply:
Thanks for double checking me.

That was weird for me the first time I came across something like that too. In the C-style languages with traditional ternary expressions, it feels a lot more natural: `return _list.length ? _list[0] : null;`

But Python changes the order when you write it, so you have the terrifying `list[0]` written first!